ingestion-beam/src/main/java/com/mozilla/telemetry/transforms/FailureMessage.java

```diff
@@ -35,6 +36,16 @@ public static PubsubMessage of(Object caller, String payload, Throwable e) {
     return FailureMessage.of(caller, payload.getBytes(StandardCharsets.UTF_8), e);
   }
 
+  /**
+   * Return a PubsubMessage wrapping a SponsoredInteraction with attributes describing the error.
+   * TODO Investigate refactoring to not import SponsoredInteraction into this higher-level module.
+   */
+  public static PubsubMessage of(Object caller, SponsoredInteraction interaction, Throwable e) {
+    PubsubMessage message = new PubsubMessage(
+        interaction.toString().getBytes(StandardCharsets.UTF_8), Map.of());
```

Comment:
We send in an empty map of attributes here, thus almost all of the fields of the error message in BQ are null. In particular, `submission_timestamp` is null, which explains why we aren't seeing these messages appearing in BQ.

Reply:
We should at minimum hydrate the attributes map with submission_timestamp, document_type, and document_version. It would be good to include geo as well.